likecoin/admin/restful.php

```diff
@@ -213,6 +388,38 @@ function () {
 					},
 				)
 			);
+			register_rest_route(
+				'likecoin/v1',
+				'/posts/(?P<id>\d+)/arweave/estimate',
+				array(
+					'methods'             => 'POST',
+					'callback'            => 'likecoin_rest_post_arweave_estimate',
+					'args'                => array(
+						'id' => array(
+							'validate_callback' => 'likecoin_is_numeric',
+						),
+					),
+					'permission_callback' => function () {
+						return current_user_can( 'manage_options' );
```

Comment:
I think `current_user_can( 'edit_post', $post_id )` is a more correct permission, since this is related to posting